common/data_refinery_common/job_lookup.py

```diff
@@ -122,6 +122,13 @@ def determine_processor_pipeline(sample_object: Sample) -> ProcessorPipeline:
     if not _is_platform_supported(sample_object.platform_accession_code):
         return ProcessorPipeline.NONE
 
+    # Optional explicit filetype checks
+    if original_file:
+        if original_file.filename[-3].upper() == "CEL":
+            return ProcessorPipeline.AFFY_TO_PCL 
+        if original_file.filename[-3].upper() == "TXT":
+            return ProcessorPipeline.NO_COP 
```

Comment:
Hey man, I don't love cops either but it'd be great if you could keep your opinions out of the code :P

However are you 100% sure that we won't have any raw data files with .txt extensions? This seem to introduce the opportunity for us to NO_OP raw data if it's ever mis-named.